libraries/AP_HAL/UARTDriver.h

```diff
@@ -15,6 +15,8 @@ class AP_HAL::UARTDriver : public AP_HAL::BetterStream {
     UARTDriver(const UARTDriver &other) = delete;
     UARTDriver &operator=(const UARTDriver&) = delete;
 
+    uint16_t _last_options;
```

Comment:
should be private

Reply:
maybe stick a comment beside it as well..

Reply:
Fixed